tests/test_protocol_greetings.rs

```diff
@@ -0,0 +1,104 @@
+use colink::{
+    extensions::instant_server::{InstantRegistry, InstantServer},
+    Participant,
+};
+use rand::Rng;
+use std::process::{Child, Command, Stdio};
+
+const USER_NUM: [usize; 11] = [2, 2, 2, 2, 2, 3, 3, 4, 4, 5, 5];
+
+#[tokio::test]
```

Comment:
Just came to my thought that, can we have a dedicated test for checking MQ connection as a testing fixture? Not sure if it's easy to achieve in Rust but maybe you can take a look?